server/src/util.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const YAML = require('yamljs');
+
+/**
+ * Fetches the input file extension
+ * @param {object} file - The input file received as form-data
+ * @returns {string} The extension of the input file
+ */
+function getFileExtension(file) {
+  if(!file.originalname) return null;
+  return path.parse(file.originalname).ext;
+}
+
+/**
+ * Parses the input specification
+ * @param {object} file - The input file received as form-data
+ * @returns {object|string} The JSON for the input specification
+ */
+function parseInputFile(file) {
+  const fileExt = getFileExtension(file);
+  let origFile = Buffer.from(file.buffer).toString();
+  if(fileExt === '.json') {
+    origFile = JSON.parse(origFile);
```

Comment:
can we use originalFile, what are you achieving by saving 4 letters? you're definitely compromising on readability